packages/react-badge/src/components/Badge/useBadgeStyles.ts

```diff
@@ -208,6 +208,24 @@ const useStyles = makeStyles({
     alignItems: 'center',
     height: '100%',
   },
+  iconTiny: {
+    fontSize: '6px',
+  },
+  iconExtraSmall: {
+    fontSize: '10px',
+  },
+  iconSmall: {
+    fontSize: '12px',
+  },
+  iconMedium: {
+    fontSize: '12px',
+  },
+  iconLarge: {
+    fontSize: '16px',
+  },
+  iconExtraLarge: {
+    fontSize: '20px',
+  },
```

Comment:
Should you be using the `tokens` from the theme instead of the raw values (there is not one for `iconTiny` but there is one for every other variant).

```suggestion
  iconTiny: {
    fontSize: '6px',
  },
  iconExtraSmall: {
    fontSize: tokens.fontSizeBase100,
  },
  iconSmall: {
    fontSize: tokens.fontSizeBase200,
  },
  iconMedium: {
    fontSize: tokens.fontSizeBase300,
  },
  iconLarge: {
    fontSize: tokens.fontSizeBase400,
  },
  iconExtraLarge: {
    fontSize: tokens.fontSizeBase500,
  },
```

Reply:
Those tokens are for text sizes; I don't think we should be using them for icon sizes. As far as I know, we don't have tokens for icon sizes, right?

Reply:
Yeah, we don't unfortunately, maybe it is something we should pass by design.